src/components/icon/icon-importer.ts

```diff
@@ -164,7 +166,9 @@ export class MynahUIIconImporter {
     TRANSFORM,
     HISTORY,
     IMAGE,
-    PIN
+    PIN,
+    DESKTOP,
```

Comment:
from where are these icons sourced from? Is there a standard library we reference, if yes can a link to that be added?